instrumentation/active_record/opentelemetry-instrumentation-active_record.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'opentelemetry-api', '~> 1.0'
   spec.add_dependency 'opentelemetry-instrumentation-base', '~> 0.19.0'
+  spec.add_dependency 'ruby2_keywords'
```

Comment:
do we need this still?

Reply:
Yep, the patches are still need to work in ruby 2.5/2.6. We can define our own ruby2_keywords, but there is some logic here to make sure we don't redefine things we don't need etc (https://github.com/ruby/ruby2_keywords/blob/master/lib/ruby2_keywords.rb)

Reply:
👍 sounds good